### ISSUE TITLE
sidebar overlay not working?

### ISSUE THREAD

**Issue by tierneyn** [poole/lanyon#147]
Hi there!

Thanks for making such an awesome theme, really loving learning about Jekyll.

The documentation says that you can 
> Make the sidebar overlap the viewport content with a single class:

```
<body class="sidebar-overlay">
  ...
</body>
```
> This will keep the content stationary and slide in the sidebar over the side content

However, adding this to the `default.html`, It seems to be the same?

My repo is here: https://github.com/tierneyn/tierneyn.github.io

**Comment by mafagafogigante** [comment]
I've done some investigation, but decided not to go any further. A difference between my website (you can check it in my profile) and yours is the presence (in mine) and the absence (in yours) of the following script

    <script>
      (function(document) {
        var toggle = document.querySelector('.sidebar-toggle');
        var sidebar = document.querySelector('#sidebar');
        var checkbox = document.querySelector('#sidebar-checkbox');

        document.addEventListener('click', function(e) {
          var target = e.target;

          if(!checkbox.checked ||
             sidebar.contains(target) ||
             (target === checkbox || target === toggle)) return;

          checkbox.checked = false;
        }, false);
      })(document);
    </script>

Note that I am **not** saying that this is the cause of the problem. It **does not** look like it. But it tells that we are clearly using different versions of lanyon (I did not write the script myself).

What I think is the cause of the issue is that your ``public/css/lanyon.css`` does not have anything on sidebar-overlay.

I get 4 lines of output for ``cat public/css/lanyon.css | grep sidebar-overlay`` with my repository, but 0 with yours. So this issue seems to be **yours**, not a problem with lanyon.


Lastly, my website does not use ``sidebar-overlay``, so you have to edit the HTML to see that it works there.

**Comment by tierneyn** [comment]
Thank you very much for that, I really appreciate you taking the time to help me.

As you suggested, it looked like my lanyon.css file was missing the code for sidebar-overlay. Indeed, it was missing: 

```
/*
 * Overlay sidebar
 *
 * Make the sidebar content overlay the viewport content instead of pushing it
 * aside when toggled.
 */

.sidebar-overlay #sidebar-checkbox:checked ~ .wrap {
  -webkit-transform: translateX(0);
      -ms-transform: translateX(0);
          transform: translateX(0);
}
.sidebar-overlay #sidebar-checkbox:checked ~ .sidebar-toggle {
  box-shadow: 0 0 0 .25rem #fff;
}
.sidebar-overlay #sidebar-checkbox:checked ~ .sidebar {
  box-shadow: .25rem 0 .5rem rgba(0,0,0,.1);
}

/* Only one tweak for a reverse layout */
.layout-reverse.sidebar-overlay #sidebar-checkbox:checked ~ .sidebar {
  box-shadow: -.25rem 0 .5rem rgba(0,0,0,.1);
}
```

Once added, this enabled the feature.

Thanks again, :)

**Comment by mafagafogigante** [comment]
I'm glad I was able to help you.

**Comment by scotv** [comment]
@mafagafogigante 

Hi, have you fix the `sidebar-overlay` issue.


I noticed these `CSS` for `sidebar-overlay`, but when I apply it on my website, the `sidebar-item` is unclickable, i entered into `F12` mode, the `<div class="wrap">` overlay on the top, that made `sidebar-item` cannot be clicked.

I checked your website, it now has been disabled for `sidebar-overlay`.

Did you have this issue before, or you just switch to `sidebar-overlay` successfully at that time, and switch off now? 

Thanks, and here is my site:  

https://github.com/scotv/blog
